frb_codegen/src/generator/c/mod.rs

```diff
@@ -1,14 +1,84 @@
-pub fn generate_dummy(func_names: &[String]) -> String {
+use std::path::Path;
+
+use crate::config;
+use crate::utils::{relative_path, BlockIndex, PathExt};
+
+pub fn generate_dummy(
+    config: &config::Opts,
+    configs: &[config::Opts],
+    func_names: &[String],
+    c_path_index: usize,
+) -> String {
+    match configs.len() > 1 {
+        true => {
+            let basic_dummy_func = get_dummy_func(&config.class_name, func_names);
+            if config.block_index == BlockIndex(0) {
+                let func_names = configs
+                    .iter()
+                    .map(|e| get_dummy_signature(&e.class_name))
+                    .collect::<Vec<_>>();
+
+                let other_headers = configs
+                    .iter()
+                    .skip(1)
+                    .map(|e| {
+                        // get directory only from paths
+                        let src_p = Path::new(&config.c_output_path[c_path_index]);
+                        let src_dir = src_p.directory_name_str().unwrap();
+                        let dst_p = Path::new(&e.c_output_path[c_path_index]);
+                        let dst_dir = dst_p.directory_name_str().unwrap();
+                        // get reletive path and head file name
+                        let relative_p = relative_path(src_dir, dst_dir);
+                        let head_file_name = dst_p.file_name_str().unwrap();
```

Comment:
uber-nit: "head" -> "header" (occurred in many places, would be great to fix all of them)

Reply:
fixed in https://github.com/fzyzcjy/flutter_rust_bridge/pull/1024/commits/cbd8da41d3f47156452430b5a2774aa471e2ddb2.